src/Utils/request/uploadFile.ts

```diff
@@ -31,7 +33,14 @@ const uploadFile = async (
         } catch {
           error = xhr.responseText;
         }
-        Notification.BadRequest({ errs: error.errors });
+        handleHttpError(
+          new HTTPError({
+            message: t("validation_error"),
```

Comment:
Also, why do we need to transalte this in the first place!? This is not shown anywhere right?